composer/profiler/torch_profiler.py

```diff
@@ -31,7 +42,7 @@ class TorchProfiler(Callback):
         pip install tensorbaord torch_tb_profiler
         tensorboard --logdir tensorboard_trace_handler_dir
 
-    Also see https://pytorch.org/docs/stable/profiler.html.
+    Also see :doc:`profiler`.
```

Comment:
Missing a "the" in the note below...it should read...

When record_shapes is specified, THE profiler will temporarily hold...

Reply:
Also docstring below is missing a tick mark around torch_profiler